src/main/java/duke/command/CommandList.java

```diff
@@ -0,0 +1,152 @@
+    private static final int CMD_TODO = 1;
+    private static final int CMD_EVENT = 2;
+    private static final int CMD_DEADLINE = 3;
+    private static final int CMD_LIST = 4;
+    private static final int CMD_DONE = 5;
+    private static final int CMD_TERMINATE = 6;
+    private static final int CMD_DELETE = 7;
+    private static final String TODO = "todo";
+    private static final String EVENT = "event";
+    private static final String DEADLINE = "deadline";
+    private static final String BY = "/by";
+    private static final String AT = "/at";
+    private static final int FIVE = 5;
+    private static final int SEVEN = 7;
+    private static final String border = "____________________________________________________________\n";
+
+    protected int command;
+    protected int taskCount = 0;
+
+    public CommandList() {
+
+        this.command = CMD_NOT_FOUND;
+    }
+
+    public void setCommand(int command) {
+        this.command = command;
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    public void loadTaskCount(ArrayList<Task> items) {
+        this.taskCount = items.size();
+    }
+
+    public void addTaskMessage(Task task) {
+        taskCount++;
+        System.out.println(border);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+        System.out.println(border);
+    }
+    public void removeTaskMessage(Task task) {
+        System.out.println(border);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+        System.out.println(border);
+    }
+
+    public void addEvent(ArrayList<Task> items, String description, String time) {
+        Events newEvent = new Events(description, time);
+        items.add(taskCount, newEvent);
+    }
+
+    public void addDeadline(ArrayList<Task> items, String description, String by) {
+        Deadlines newDeadline = new Deadlines(description, by);
+        items.add(taskCount, newDeadline);
+    }
+
+    public void addTodo(ArrayList<Task> items, String description) {
+        Todo newTodo = new Todo(description);
+        items.add(taskCount, newTodo);
+    }
+    public void removeItem(ArrayList<Task> items, int taskNum) {
+        taskCount--;
+        removeTaskMessage(items.get(taskNum));
+        items.remove(taskNum);
+    }
+
+    public static void printEndMessage() {
+        System.out.println(border);
+        System.out.println("chat again next time!\n" + border);
+    }
+
+    public void printErrorMessage(DukeException error) {
+        error.printError(error.getErrorType());
+    }
+
+    public void executeCommand(ArrayList<Task> items, DukeException error, String input) {
+        String[] inputs;
+        switch (command) {
+            case CMD_TODO:

```

Comment:
This deviates from the switch case statement [layout conventions ](https://se-education.org/guides/conventions/java/basic.html#layout)- indentation for `case` is not needed.